README.md

```diff
@@ -25,230 +25,53 @@ Freshli reads dependency information from special files called "dependency manif
   * composer - reads information from `composer.json` and `composer.lock`
 * Python
   * pip - reads information from `requirements.txt`
+* .NET
+  * NuGet - reads information from `*.csproj`
 
 ### Source Code Repositories
 
 Freshli reads source code repository history to access previous version of each dependency manifest. These are the source code repositories that it currently works with. If you don't see your favorite, create an issue for us to add support for it.
 
 * Git
 
-## Getting started with `freshli` CLI
+## Using Freshli in a Project
 
-### Running `freshli`
+To use Freshli in a project, you have two options:
 
-To run `freshli` from the command line, the project [freshli-cli](https://github.com/corgibytes/freshli-cli) is required.
-Please reference the [README](https://github.com/corgibytes/freshli-cli#getting-started-with-freshli-cli) instructions found there.
+### Stable and Beta Releases
 
+Stable and beta releases are found at https://www.nuget.org/packages/Freshli/ and can be added with the [`dotnet add package`](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-add-package) command:
```

Comment:
No point in adding this to the ReadMe until the package exists.  Let's try for a beta in  [v0.4.0](https://github.com/corgibytes/freshli/milestone/5).